src/locutil.rs

```diff
@@ -0,0 +1,39 @@
+// TODO import lineBreakG fn from whitespace
```

Comment:
I guess a nice TODO format would be:

```
TODO(Harry): do something
```

Reply:
ig it's pretty obvious that i have to do it.

Reply:
I mean, overtime there will be too many TODOs in the codebase, it'd be helpful to see everyone's TODOs separately.